docs/requirements.txt

```
@@ -1,3 +1,3 @@
-sphinx==4.0.1
+sphinx==4.2.0
 sphinx-rtd-theme
```

@algochoi With the _suggestion_ below, I think the PR is ready for approval.

https://github.com/algorand/pyteal/pull/279 is the corresponding change set that fixed the PyTeal readthedocs build.  Thanks for reflecting `.readthedocs.yaml` into the PR.

```suggestion
sphinx-rtd-theme==1.0.0
```

---

Thanks for the catch - will commit the suggestion